### PR TITLE
Feature vnctl static translations

### PR DESCRIPTION
**Pull request by toros11**
see #429

### CODE REVIEW

**Comment by rakshasa** [comment]
+1

File: client/vnctl/lib/vnctl/cli/base.rb [hunk 1]
```diff
@@ -242,7 +242,7 @@ def self.define_mode_relation(mode_type, required_opts = [], &block)
           }
 
           yield self if block_given?
-          desc "del #{base_uuid.upcase}_UUID" , "Removes a(n) #{mode_type} #{base_uuid}."
+          desc "del #{base_uuid.upcase}_UUID --option=OPTION", "Removes a(n) #{mode_type} #{base_uuid}."
```
Comment (Metallion) [inline client/vnctl/lib/vnctl/cli/base.rb]: What is the `--option=OPTION` for? I don't think this command takes an --option argument.

```bash
> vnctl translations static add tr-q5z00rzz --ingress-ipv4-address 10.0.0.1 --egress-ipv4-address 10.0.0.5 --option joske

ERROR: vnctl add was called with arguments ["tr-q5z00rzz", "--option", "joske"]
Usage: "vnctl translations static_address add TRANSLATION_UUID --option=OPTION --egress-ipv4-address=EGRESS_IPV4_ADDRESS --ingress-ipv4-address=INGRESS_IPV4_ADDRESS".
```

Comment (rakshasa) [inline client/vnctl/lib/vnctl/cli/base.rb]: If anything it should have been '--option foo'.

More of an issue is that we don't really make it clear in the help text how to pass additional options, and '--foo=bar' is much more common in cli than '--foo bar'.

Should probably not be in this pull request.

Comment (Metallion) [inline client/vnctl/lib/vnctl/cli/base.rb]: That's one way of doing it but for consistency's sake, I think it's better to stay with the `[OPTIONS]` way of writing it. That's how it's currently written everywhere else in vnctl.

**Pull request merged**